recipes/modern-xorg-stack/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "modern-xorg-stack" %}
```

Comment:
Probably; I'm certainly not wedded to the current name. I definitely like the metapackage name of `xorg-libs` used by @gqmelo's recipes.